components/resolved_ts/src/advance.rs

```diff
@@ -106,10 +104,6 @@ impl AdvanceTsWorker {
         let pd_client = self.pd_client.clone();
         let scheduler = self.scheduler.clone();
         let timeout = self.timer.delay(advance_ts_interval);
-        let min_timeout = self.timer.delay(cmp::min(
-            DEFAULT_CHECK_LEADER_TIMEOUT_DURATION,
-            self.advance_ts_interval,
```

Comment:
The `self.advance_ts_interval` is not used any more it could be removed from the struct.